scripts.js

```diff
@@ -1 +1,89 @@
-console.log('Test');
\ No newline at end of file
+// First task:
+// Add up all of the salaries
+const salaries = {
+    john: 50,
+    adam: 100,
+    katie: 75
+}
+const salarySum = salaries.john + salaries.adam + salaries.katie;
+console.log(salarySum);
+
+// Second task:
+// Write the isObject() function. Use typeof operator.
+function isObject(valueToCheck) {
+    return typeof valueToCheck === "object" && valueToCheck !== null;
```

Comment:
Why do you need `&& valueToCheck !== null;`

Reply:
Right it is useless in this function